mne/viz/evoked.py

```diff
@@ -185,7 +185,7 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
     titles = _handle_default('titles', titles)
     units = _handle_default('units', units)
     # Valid data types ordered for consistency
-    channel_types = ['eeg', 'grad', 'mag', 'seeg', 'dipole', 'gof']
+    channel_types = ['eeg', 'grad', 'mag', 'eog', 'seeg', 'dipole', 'gof']
```

Comment:
You should be able to add another kwarg like plot_types that defaults to
what channel_types used to be. You could add an 'all' convenience shortcut
option to plot all available types.